Readme.md

```diff
@@ -0,0 +1,81 @@
+📗 Table of Contents
+    📖 About the Project
+        🛠 Built With
+            Tech Stack
+
+            🚀 Live Demo
+    💻 Getting Started
+        Setup
+        Prerequisites
+        Install
+        Usage
+        Run tests
+    
+    👥 Authors
+    🔭 Future Features
+    🤝 Contributing
+    ⭐️ Show your support
+    🙏 Acknowledgements
+    📝 License
+
+
+📖 [HELLO  MICROVERSE]
+
+Hello microverse is a program that displaysa microverse greeting page.
+
+🛠 Built With
+
+    Tech Stack
+
+    Client
+     --HTML
+     --CSS
+
+
+💻 Getting Started
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder:
+
+Usage
+To run the project, execute the index.html file.
+
+
+
+
+👤 Author
+
+GitHub: @fredo509
+Twitter: @fredo_st_fleur
+LinkedIn: Fredo st fleur
+
+
+🔭 Future Features
+- Add  "Say Hello" button
+
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+
+
+⭐️ Show your support
+    If you like this project please give me a star .
+
+🙏 Acknowledgments
+    I would like to thank mmicroverse for this beautiful project
+
+
+📝 License
+    This project is MIT licensed.
+
+
+
+
```

Comment:
- [ ]  According to this provided [Readme Template ](https://github.com/microverseinc/readme-template) kindly use it to personalize your project readme to make it more professional and try to maintain its schema to help people who visit your repo to not get confused  if it is header let it be a header or if its bullet points let it be that way this will make it more clear